docs/environment-variables.md

```diff
@@ -70,3 +71,48 @@ FLASK_ENVIRONMENT_CONFIG=dev
 SECRET_KEY='some_random_key'
 (...)
 ```
+
+# Database Environment Variables
+
+To run the backend in other than `local` mode i.e `prod` or `develop` you need to export database environment variables.
+These are the required database environment variables.
+```
+export DB_TYPE=<database_type>
+export DB_USERNAME=<database_username>
+export DB_PASSWORD=<database_password>
+export DB_ENDPOINT=<database_endpoint>
+export DB_NAME=<database_name>
+```
+ ## Database environment variables description
+
+
+| Environment Variable | Description                                                                 | Example           |
+|----------------------|-----------------------------------------------------------------------------|-------------------|
+| DB_TYPE | Type of database you want to use | postgres, mysql
+| DB_USERNAME          | Username of the user through which you will be doing operations in database | admin123, jhonDoe | 
+| DB_PASSWORD | Database password of the `DB_USERNAME` user| admin@123, mypwd123|
+| DB_ENDPOINT | Path to database or connection string connecting to database | /path/to/database.db |
+| DB_Name | Name of the database to which you want to connect| Users.db, Tasks.db
```

Comment:
- `DB_NAME` instead of `DB_Name`
- these variable examples are not correct since it is not about the table name, but the database name. So for example, if you use a service like ElephantSQL for example. So as examples maybe you could have `database1`? It's hard to come up with an easy to understand example 🤔